src/Symfony/Component/Messenger/Tests/Adapter/AmqpExt/Fixtures/long_receiver.php

```diff
@@ -27,13 +27,15 @@
 );
 
 $connection = Connection::fromDsn(getenv('DSN'));
-$sender = new AmqpSender($serializer, $connection);
 $receiver = new AmqpReceiver($serializer, $connection);
 
 $worker = new Worker($receiver, new class() implements MessageBusInterface {
     public function dispatch($message)
     {
         echo 'Get message: '.get_class($message)."\n";
+        if ($message instanceof ConfiguredMessage) {
+            echo sprintf("with configs: %s\n", json_encode(array_keys($message->allConfigurations()), JSON_PRETTY_PRINT));
```

Comment:
Why do you use `json_encode` here? IMHO better use standard `serialize` because configurations implements `Serializable`, not `JsonSerializable`.

Reply:
This is just for tests purpose anyway :)

Reply:
Done